content/en/docs/contribute/generate-ref-docs/kubectl.md

```diff
@@ -152,65 +135,62 @@ milestone in your pull request. If you don’t have those permissions, you will
 need to work with someone who can set the label and milestone for you.
 {{< /note >}}
 
-## Editing Makefile
+## Setting build variables
 
-Go to `<rdocs-base>`, and open the `Makefile` for editing:
+Go to `<rdocs-base>`. On you command line, set the following environment variables.
 
-* Set `K8SROOT` to `<k8s-base>`.
-* Set `WEBROOT` to `<web-base>`.
-* Set `MINOR_VERSION` to the minor version of the docs you want to build. For example,
-if you want to build docs for Kubernetes 1.15, set `MINOR_VERSION` to 15. Save and close the `Makefile`.
+* Set `K8S_ROOT` to `<k8s-base>`.
+* Set `WEB_ROOT` to `<web-base>`.
+* Set `K8S_RELEASE` to the version of the docs you want to build.
+  For example, if you want to build docs for Kubernetes 1.17, set `K8S_RELEASE` to 1.17.
 
-For example, update the following variables:
+For example:
 
-```
-WEBROOT=$(GOPATH)/src/github.com/<your-username>/website
-K8SROOT=$(GOPATH)/src/k8s.io/kubernetes
-MINOR_VERSION=15
+```shell
+export WEB_ROOT=$(GOPATH)/src/github.com/<your-username>/website
+export K8S_ROOT=$(GOPATH)/src/k8s.io/kubernetes
+export K8S_RELEASE=1.17
 ```
 
-## Creating a version directory
+## Creating a versioned directory
 
-The version directory is a staging area for the kubectl command reference build.
-The YAML files in this directory are used to create the structure and navigation
-of the kubectl command reference.
+The versioned directory is a staging area for building the kubectl command reference.
+The `createversiondirs` build target creates the versioned directory.
+This step ensures that a copy of the configuration files are created in
+the versioned build directory. The configuration may require updates for
+a new release.
+The versioned directory name follows the pattern of `v<major>_<minor>`.
 
-In the `<rdocs-base>/gen-kubectldocs/generators` directory, if you do not already
-have a directory named `v1_<MINOR_VERSION>`, create one now by copying the directory
-for the previous version. For example, suppose you want to generate docs for
-Kubernetes 1.15, but you don't already have a `v1_15` directory. Then you could
-create and populate a `v1_15` directory by running these commands:
+In the `<rdocs-base>` directory, run the following build target:
 
 ```shell
-mkdir gen-kubectldocs/generators/v1_15
-cp -r gen-kubectldocs/generators/v1_14/* gen-kubectldocs/generators/v1_15
+make createversiondirs
 ```
 
 ## Checking out a branch in k8s.io/kubernetes
```

Comment:
May want to change this to `Checking out a release tag in k8s.io/kubernetes

Reply:
Good idea